### PR TITLE
README.md 

### LINKED ISSUE #1: Create Readme File planning

### PR DESCRIPTION
**Pull request by firity**

### CODE REVIEW

File: planning/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Planning
+
+## Group Rules HYF-Group2
+
```
Comment (yildiztugba) [inline planning/README.md]: Hi :blush: ,Can you add group rules? for example: finishing taks on time, informing team member about what s/he did, help each other, being respectful etc.

File: planning/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+| Wednesday |           |              |                                         |
+| Thursday  |           |              |                                         |
+| Friday    |           |              |                                         |
+| Saturday  |           |              |                                         |
+
+## Communication Channels
+
+how often will we get in touch on each channel, and what we will discuss there:
+
+- **Slack Messaging**:
```
Comment (yildiztugba) [inline planning/README.md:20]: can you fill in this part too?

**Review by yildiztugba — CHANGES_REQUESTED** [review]
Hi :blush:, I saw some parts missing, can you add them please?
Also, please add label,milestone, project,assign and linked pr with issue.

**Comment by yildiztugba** [comment]
your issue is also to create an empty file but i just reviewed :hand_over_mouth: maybe you can change your titles (both issue and pr)

File: planning/README.md (hunk 2 above)
Comment (firity) [reply planning/README.md:20]: i can fill that after our meetings

File: planning/README.md (hunk 1 above)
Comment (firity) [reply planning/README.md]: okey noproblem i will add them

Comment (yildiztugba) [reply planning/README.md]: ok, thank you :hugs:

**Review by yildiztugba — APPROVED** [review]
:+1:  thank you for changes

**Pull request merged**